### PR TITLE
package.json: Drop unused cssnano-preset-lite, fix stylelint errors on main

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

**Review by jelly — DISMISSED** [review]

File: .stylelintrc.json [hunk 1]
```diff
@@ -4,8 +4,6 @@
     "stylelint-use-logical-spec"
   ],
   "rules": {
-    "indentation": 2,
-
```
Comment (jelly) [inline .stylelintrc.json:8]: This seems suspicious? Why is it dropped?

Comment (martinpitt) [reply .stylelintrc.json:8]: https://stylelint.io/migration-guide/to-15 and https://stylelint.io/user-guide/rules/indentation/ just say that it's deprecated, but not why :man_shrugging:

Comment (jelly) [reply .stylelintrc.json:8]: Ugh, well as the default is 2 anyway. LGTM.

Comment (martinpitt) [reply .stylelintrc.json:8]: Oh, that could be? They want to stop supporting anything else than 2?

**Review by jelly — APPROVED** [review]

**Pull request merged**